server/prisma/migrations/20240701151138_init/migration.sql

```diff
@@ -0,0 +1,9 @@
+-- CreateTable
+CREATE TABLE "Post" (
```

Comment:
Looks like what happened here was that originally in schema.prisma you had the model named as Post, then ran a migration. And then later you went and changed the model in the schema.prisma file to User. Since you changed your schema again you should delete this migration folder and run a new migration to reflect your changes.